### PR TITLE
8.2.5

### PR DESCRIPTION
**Pull request by imonroe**
# READY FOR REVIEW

# Summary
- Some small tweaks to basic page plus functionality from feedback from Ishi.

# Needed By (Date)
- before 2.1.0 deployment.

# Urgency
- medium

# Steps to Test

1. Do this
1. Then this
2. Then this

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: config/sync/core.entity_form_display.node.stanford_page.default.yml [hunk 1]
```diff
@@ -90,14 +89,10 @@ content:
     region: content
     third_party_settings: {  }
   su_basic_page_type:
-    type: options_shs
+    type: options_select
```
Comment (pookmish) [inline config/sync/core.entity_form_display.node.stanford_page.default.yml:92]: Why this change?

File: config/sync/layout_library.layout.stanford_basic_page_full.yml [hunk 2]
```diff
@@ -59,6 +59,36 @@ layout:
...
+      extra_classes: ''
+      centered: centered-container
+      columns: default
+      label: 'page description'
+    components:
+      5659aa07-22e4-461e-b826-00d505ba2d0c:
+        uuid: 5659aa07-22e4-461e-b826-00d505ba2d0c
+        region: main
+        configuration:
+          id: 'field_block:node:stanford_page:su_page_description'
```
Comment (pookmish) [inline config/sync/layout_library.layout.stanford_basic_page_full.yml:74]: Ooh we missed that on the full width page.

**Review by pookmish — COMMENTED** [review]
Should we just commit this to the dev branch incase any additional hotfixes come through?

File: config/sync/views.view.stanford_basic_pages.yml [hunk 3]
```diff
@@ -572,7 +572,7 @@ display:
           relationship: none
           group_type: group
           admin_label: ''
-          default_action: ignore
+          default_action: empty
```
Comment (pookmish) [inline config/sync/views.view.stanford_basic_pages.yml:575]: I know the reasoning for this, but it also is confusing to a user that doesn't input any argument into the paragraph data field. So there's a trade off with this for the content author

File: config/sync/core.entity_form_display.node.stanford_page.default.yml (hunk 1 above)
Comment (imonroe) [reply config/sync/core.entity_form_display.node.stanford_page.default.yml:92]: Ishi found the simple hierarchal select was confusing, in that it appeared to the inexperienced user to apply two or more terms if you choose a term below the root level of the taxonomy.  By changing it to a simple select, it's more obvious that only one term is being applied.

File: config/sync/layout_library.layout.stanford_basic_page_full.yml (hunk 2 above)
Comment (imonroe) [reply config/sync/layout_library.layout.stanford_basic_page_full.yml:74]: Yeah, it was tricky to remember that the layout isn't the same between default and full-width, but I think I got it right here.

File: config/sync/core.entity_form_display.node.stanford_page.default.yml (hunk 1 above)
Comment (pookmish) [reply config/sync/core.entity_form_display.node.stanford_page.default.yml:92]: i guess that kinda makes sense but could also be resolved with help text. nothing i have a strong opinion about though.

File: config/sync/views.view.stanford_basic_pages.yml (hunk 3 above)
Comment (imonroe) [reply config/sync/views.view.stanford_basic_pages.yml:575]: Yeah, this is another thing Ishi and I went over in a call.  It doesn't make sense to show all basic pages, and since you can use multiple terms, you can construct the list how you like.  It's a documentation thing to make sure the users know how to add the filter.

File: config/sync/layout_library.layout.stanford_basic_page_full.yml (hunk 2 above)
Comment (pookmish) [reply config/sync/layout_library.layout.stanford_basic_page_full.yml:74]: Did it have to be a whole other layout? could it just be added to the same section as the page title? it's just a little less confusing when editing.

File: config/sync/views.view.stanford_basic_pages.yml (hunk 3 above)
Comment (pookmish) [reply config/sync/views.view.stanford_basic_pages.yml:575]: If we get user's commenting on it, we can always adjust as necessary.

File: config/sync/layout_library.layout.stanford_basic_page_full.yml (hunk 2 above)
Comment (imonroe) [reply config/sync/layout_library.layout.stanford_basic_page_full.yml:74]: I made it match the way it was set up in the default layout, and they both use the same theming. Do you mean confusing when editing the layout, or confusing for content authors?

Comment (pookmish) [reply config/sync/layout_library.layout.stanford_basic_page_full.yml:74]: Yeah the result is the same. i don't really care in the long run. 1 layout with 2 components vs 2 layouts with 1 component each. it's still the same outcome.

Comment (pookmish) [reply config/sync/layout_library.layout.stanford_basic_page_full.yml:74]: So let's remove this and let's also remove the description from the regular node display.

Comment (imonroe) [reply config/sync/layout_library.layout.stanford_basic_page_full.yml]: This is fixed.  Page description removed from both the default and full page layout.

**Review by pookmish — APPROVED** [review]
